### DISCUSSION TITLE
Why should I farm?

### DISCUSSION THREAD

**Discussion by xLaBJup** [Chia-Network/chia-blockchain#1427] [Q&A]
Hello everybody from a newbie.

I have managed to install two plots of k=32 on a laptop (HDD - one plot at a time, took a bit). Now the farm dashboard indicates "Time to win = 5 months" (to win a block of two XCH, right?). 

Now depending on the expected value of a XCH coin, I was wondering: **Is farming worth at all the "effort" for a "mini-farmer"?** 
As, moreover, there is no guarantee (as it is a lottery), my impression is that the "target audience" as farmers are folks who are willing to invest in loads of TB harddrives (or do already have so) to have any chance of harvesting anything significant at all. 

Why would it be interesting at all for folks "who just happen to have some disk space", to make that available? 
Why would a block of 2 XCH over 5 months be an incentive?

Thank you for any thoughts on this.

Best regards!

**Comment by dorkmo** [comment]
if chia ever becomes as valuable as bitcoin, itd be nice to have 2 chia

**Reply by GeorgePapageorgakis** [reply]
And why wait for chia when there are already so many options out there?
I ve been wondering my self that question too. If I have to spent 5000$ for hdds and ssds (that will probably be destroyed in the process with so many terrabytes of writting) is it worth it? The algorithm does not scale fairly at all. If you need days to get a chia in the start imagine in a couple of months ahead. Only those who own hdd rigs and mini datacenter will be able to farm some...

Is there a chance to rebalance reward process?

**Reply by hoffmang9** [reply]
The rewards are directly proportional to the space you have plotted versus all space plotted. There is no fairer way known to the current state of the art.

**Reply by xLaBJup** [reply]
> The rewards are directly proportional to the space you have plotted versus

So a "big player" might as well set up a datacenter and make sure to get the biggest share of XCH ?
The ones with the biggest amount of ressources will gather the win - to caricature: "The Top 5% take 95% of the profit".

To me that looks a bit contrary to the idea of decentralization.

> What was intended to be a decentralized network is now controlled by a small number of miners (...)

one could as well replace "miners" by "farmers" in that sentence, no?

**Reply by GeorgePapageorgakis** [reply]
> 
> 
> The rewards are directly proportional to the space you have plotted versus all space plotted. There is no fairer way known to the current state of the art.

Havent you applied any diminishing returns on the space that someone can have? A soft cap or something. This is not decentralization...

**Reply by hoffmang9** [reply]
There is no way to do what you're asking for because then the big player splits into two soft caps. Sybill problems are real.

Other thred - "big player" isn't who you think it is. Google/Amazon/FB/Apple/MS don't have spare space. They buy by the shipping container and rack space 24/7 just in time.

**Reply by dorkmo** [reply]
just need to hijack a shipping container, then i can really start farming

**Reply by scotchegg** [reply]
What dictates the value of XCH? Bitcoin started out at ad pennies right? So if your getting 4p? 

Don't mean to be skeptical, I am not and fully intend to support the project. Just trying to understand the perceived value offering.

**Reply by dorkmo** [reply]
Once chia gets listed on big crypto markets the hope is that people will quickly see the advantages versus other coins. If we think chia is better than bitcoin and etherium, we'd then assume its worth more lol

**Reply by wanghongqi8548488** [reply]
> 奖励与绘制的空间与绘制的所有空间成正比。当前的技术水平尚无更公平的方法。

What is the reason why the software shuts down automatically when drawing?

**Comment by xLaBJup** [comment]
Thank you @hoffmang9 for taking the time to answer.

> Other thread - "big player" isn't who you think it is. Google/Amazon/FB/Apple/MS don't have spare space. 

Maybe the term was badly chosen. 
Still, as I said, it looks to me like: _"The ones with the biggest amount of resources will gather the win - to caricature: 'The Top 5% take 95% of the profit'."_

@hoffmang9 some questions:

1. Who would you think is/should be the "average farmer" in terms of available infrastructure resources? Who is "Chia's target audience"?
2. How does CHIA mitigate the risk of centralization due to the linear proportionality between available storage space (hence resources) and wins? (I know about the farm pool idea - but it is unclear in how far this will make farming interesting for small farmers)
3. As stated above and unanswered: _"Why would it be interesting at all for folks 'who just happen to have some disk space', to make that available?"_
4. As stated above and unanswered: _"Why would a block of 2 XCH over 5 months be an incentive?"_

**Reply by dorkmo** [reply]
i think the main issue is that chia was always fundamentally designed to be run and favor people with existing empty disk space. The biggest winners will continue to be people that match this discription.

**Reply by hoffmang9** [reply]
There is an unstated assumption hiding in a lot of this thread and it is "little farmers should win rewards everyday." Right now there is no reward for running a Bitcoin full node. In Chia you can take some of your empty space, plot it, and win - probably the value of your computer - during that computer's life. 

That is a fundamental change.

**Reply by xLaBJup** [reply]
There is no such assumption in my questions. That, in turn, is an assumption you make @hoffmang9 .

_"(...), plot it, and win - probably the value of your computer - during that computer's life"_ is a somewhat clearer indication to questions 3. and 4. 

Pity there are no straightforwards answers. Also to Questions 1. and 2.
Currently, 20% of winning farmers took 80% of won blocks...

![image](https://user-images.githubusercontent.com/81095428/112164500-37d33d00-8bee-11eb-8edc-cdeebfed50a2.png)

**Reply by dorkmo** [reply]
ive got about 0.4 chia per TB ive had online so far, i'd assume everyone is seeing similar numbers. Once it hits 2 chia per TB id assume most people that have 2TB would have won.

**Reply by GeorgePapageorgakis** [reply]
This is worse than farming bitcoin lol...

**Reply by dorkmo** [reply]
About the top rewards? Im sure bitcoin started off with a handful of miners getting most of the coins. Theoretically more small farmers will come online soon.

**Reply by GeorgePapageorgakis** [reply]
> 
> 
> There is an unstated assumption hiding in a lot of this thread and it is "little farmers should win rewards everyday." Right now there is no reward for running a Bitcoin full node. In Chia you can take some of your empty space, plot it, and win - probably the value of your computer - during that computer's life.
> 
> That is a fundamental change.

The problem here is that you compare bitcoin which has been farmed for a decade now, with a fresh start on a new coin... 
I bet you will adjust the issue in the future after all the early "whales" have pumped their bags... And also, you cannot compare btc to chia...

**Comment by keliew** [comment]
If you're 'investing' more than what you can afford to lose, then you should stop farming.

And this would include unused diskspace which you can forego.

Just imagine you bought a TB external disk (due to price factor), but you've only used 200GB of it, you have 800GB to spare to which you know you'll never use the most of it. You plot 2 x k32, farm, and get rewarded for the unused space. What's your lost? None. You may in fact gain some XCH over time, and if the market value hits a favourable value, your unused space in the past is rewarded handsomely. What could be a problem in this scenario?

If you wanna think towards making it profitable...then you aren't considering what the whales (with vested business interest) would need to financially and operationally commit to. They would be much more worried than your few plots of diskspace. To says that they shouldn't receive the rewards due to their investment is rather unfair. If it's about ROI, then you're thinking way too simple.

**Reply by xLaBJup** [reply]
Thank you @keliew for your comment.
You are describing exactly what I did - had some Space on a laptop left, turned it to 2 plots.
If my calculations are right, with 135 PB on mainnet, I could expect a block in about 67 days (which is also just probability, no warranty).
Question is: What do you call "rewarded handsomely"? 
@hoffmang9 mentionned "win - probably the value of your computer - during that computer's life". Of course everybody needs to assess, whether this is "handsomely".
I am not necessarily talking about economical ROI - which is why i asked about "incentive" ...
I also did not say that whales should not receive the rewards due to their investment. I asked how the risk of centralization is mitigated. 
Please tell me more about less simple thinking - I am asking for just that.

**Reply by mobeyond** [reply]
Ally to beat whales? Why is there no POOL for miners/famers?

**Comment by GeorgePapageorgakis** [comment]
![εικόνα](https://user-images.githubusercontent.com/9392917/112209842-9ca88a80-8c22-11eb-9e80-086ddff1bf74.png)
How many TB do they have the top 3 users? Do they have data centers? lol
They get everything and I get 0 even though  i have surpassed the Expected Time to Win by a factor of 2... GREAT

**Reply by Firdy999** [reply]
From my point of view If your expected time to win more than 1 day, that is not expectation this kind of wish and may wishes come true.

**Reply by GeorgePapageorgakis** [reply]
is there a calculator to find your estimated time to win?

**Reply by xLaBJup** [reply]
https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#i-have-only-10-tb-will-i-ever-win-xch-on-mainnet 
estimated time is proportional to the ratio of disk space you put to farming compared to the overall available disk space.

**Reply by xLaBJup** [reply]
> From my point of view If your expected time to win more than 1 day, that is not expectation this kind of wish and may wishes come true.

If my calculation is more or less right, with 135 PB on the mainnet as per today, one needs some 135 plots to be able to expect a XCH/day

**Reply by GeorgePapageorgakis** [reply]
So when it says 2 days estimation and I havent won a single XCH coin in almost 7 days what does this tells us?

**Reply by hoffmang9** [reply]
It tells us that you are unlucky. I was unlucky for 4 days too.

Much more here - https://chiadecentral.com/will-i-win-chia-with-2-tb-of-plots/

**Reply by xLaBJup** [reply]
It tells us, that it is a lottery, as announced ;-)

**Reply by GeorgePapageorgakis** [reply]
so many TB of writing on my poor ssd it will eventually blow up. I ll give farming a couple of weeks and see how it goes. If its still 1 coin per week well... maybe next year :D

**Comment by GeorgePapageorgakis** [comment]
I think you have to update the calculation formula of Expected Time to Win. Its completely inaccurate.

**Reply by xLaBJup** [reply]
Please enlighten me!

**Reply by GeorgePapageorgakis** [reply]
I mean in the app, if it displays 2 days expected time to win and you dont get any within a week, well seems like you could make it at least 7 days :)

**Reply by dorkmo** [reply]
ive had what seemed like 3 in a row, then a gap of multiples times the expected. Average time definately uneven.

**Comment by scotchegg** [comment]
My concern with this V traditional mining such as Eth at the moment is you always get a return to match that of your % in contribution. Where as with this method you could easily allcoate a years with of time, hardware and effort and get nothing to show for it. Nothing at all. Especilly as the % contributio gets smaller as more enter the field in coming months to years. Where is the reward for early supporters? Why not reward all with a % of their contribution 0.002% etc.

**Reply by GeorgePapageorgakis** [reply]
Well because they designed in that way, maybe there is a reason but I am still getting 0 XCH with 10 days of farming (expected time to win is shown as 2, sometimes 3)... 
So its like i am doing a hole in the water-> better not farm at all and make the coin centralized to death

People in not as wealthy countries as USA etc, cannot afford buying 1petabyte of hdds in order to get some coins. 

How hard is it to make rewards of XCH in floating/double values?

**Reply by VaulB** [reply]
I just got into this a week ago. About 120 plots farming now and no rewards. I think this might be a significant issue for the whole idea of Chia. It seems, that there is a kind of threshold amount of plots under which there is no point of wasting your time and energy to set up the farming. It would be better to earn at least a fraction of Chia in a week than no Chia at all in several weeks or even months. I think this is somewhat psychological, but it might matter a lot. For me it seems, that the threshold is already in some tens of terabytes and we have barely just begun. I think people with less than some tens of terabytes will be dropping out once they don't see rewards in weeks. New small farmers might be discouraged also, so the real unused empty storage might not be joining to this. I would argue, that at the moment most of the storage used for plotting is mainly purchased for plotting purpose only. In comparison. I could invest 10k into Ethereum mining rig and earn over 1k/month right now for sure. But, things are changing fast and I am still willing to bet on Chia.

**Reply by dorkmo** [reply]
in a few weeks they plan on releasing a pool mining solution where you can split rewards with others. Potentially allowing for smaller rewards more often.

**Reply by VaulB** [reply]
That's good news. I hope it will set the bar low enough for small farmers.

**Comment by ardabro** [comment]
Why not limit the maximum amount of space per node to let's say 1TB? It would definitely favour small players and network dispersion.

**Reply by hoffmang9** [reply]
Then all the big farmers would just be 1TB per. Sybil attacks are a thing - https://www.geeksforgeeks.org/sybil-attack/#:~:text=Sybil%20Attack%20is%20a%20type,authority%2Fpower%20in%20reputation%20systems.

**Reply by ardabro** [reply]
But now the influence on the network depends on capacity only right? So if they divide the security level won't suffer (the same capacity) but the whole operation will be more troublesome and expensive for them. So dispersion level should be higher.

**Reply by hoffmang9** [reply]
It will not. Creating trivial to defeat "security" features is not worth doing. One of the major points of Nakamoto consensus is to have some sort of solution to this problem. See page 3 of the BTC Whitepaper - https://bitcoin.org/bitcoin.pdf

**Reply by ardabro** [reply]
...and You have it pointed-out in Your papers that the idea failed and Bitcoin is centralized as hell :) What are Your premises that this project won't go the same way? I don't see any. Just buy 1 million HDDs instead of 100000 RTXes and You don't need to worry so much about Your electricity bills. BTW: players like Google, MS, FB, Amazon could easily kill the whole system already having hundreds millions of drives.

**Reply by dorkmo** [reply]
#1 farmer is winning about 5% of coins. Will have to watch and see if this grows or shrinks. https://www.chiaexplorer.com/blockchain/address/xch1f0ryxk6qn096hefcwrdwpuph2hm24w69jnzezhkfswk0z2jar7aq5zzpfj

**Reply by ardabro** [reply]
@dorkmo So how the remaining 95% are created? Whitepaper says that:
_Farming rewards will create new chia once Chia Network’sblockchain is launched. Our farming rewards schedule was directly patterned after the Bitcoin rewards schedule._
...except the fact, that:
_The Company expects to create 21 million chia at mainnet launch (Chia Network’s StrategicReserve or pre-farm) and place those on our balance sheet._
...so - according to the schedule - after 21 years of farming they will still have half of all coins :)

**Reply by hoffmang9** [reply]
At year 6 the Company will have only 58% if and only if it hasn't dividended coins to shareholders or otherwise used them to advance adoption.

**Reply by dorkmo** [reply]
#1 guy is up to 6% from 5% of winning plots today. if he gets to 10% i'm going to start worrying. also no telling if they have more than one node running.

**Reply by Firdy999** [reply]
> #1 guy is up to 6% from 5% of winning plots today. if he gets to 10% i'm going to start worrying. also no telling if they have more than one node running.

I think this not a guy, this is a pool and there is unofficial pools

**Reply by dorkmo** [reply]
7% today, at this rate they'll be over 50% in a month or two lol

need to get up and running on exchanges asap to get more people plotting

**Reply by dorkmo** [reply]
8% this morning

**Reply by geraldneale** [reply]
@dorkmo Right now address rank#1 has 6958/140218 = 4.9% not even close to 8%

6958 Coins:
https://www.chiaexplorer.com/blockchain/address/xch1f0ryxk6qn096hefcwrdwpuph2hm24w69jnzezhkfswk0z2jar7aq5zzpfj

140218 Total Coins:
https://www.chiaexplorer.com/blockchain/coins

What are you looking at?

**Reply by dorkmo** [reply]
I've just been looking at "Farmer Status Summary Won 8.37% of the last 10,000 blocks"
I feel like that gives a pretty good view of how fast they are adding plots in comparison to everyone else. The ability to add 1% more to the total network plots in a day seems pretty extreme.

**Reply by geraldneale** [reply]
Agreed extreme, but what are alternative explanations? Are you saying that the next coinbase has some sort of tendency to return to an address within the last most recent number of coinbases? That would be teased out by looking at how much a percentage of the the most recent 10,000 coinbases are given out to the smallest plots making up 50% of total plots. If much less than the expected 50% then there might be a stickiness to earning a coinbase. I don't know how to find those numbers in Chia Explorer or elsewhere however.

**Reply by GeorgePapageorgakis** [reply]
Farming algorithm just favors whales. We cant do anything since those guys have invested 50-100K $ in hdds and ssds... I quit.

**Reply by dorkmo** [reply]
looks like they might be adding a bit over a PB a day. What would that cost?

**Reply by dorkmo** [reply]
Aw! as @toadzhou pointed out in  #1719 , https://hpool.com/ seems to be the mystery address. Their website shows about 30PB and growing. chia explorer shows about 11.5% of last 10,000.

**Reply by GeorgePapageorgakis** [reply]
![εικόνα](https://user-images.githubusercontent.com/9392917/113771826-90492500-972c-11eb-991e-bfe69de974a5.png)

**Reply by keliew** [reply]
Interestingly, when total netspace (blue) dropped, it didn't make a significant dent to hpool (green). When total netspace increase drastically, hpool's plot size didn't increase accordingly...

It seems that our attention on hpool is rather...misleading.

It sure seems the number of users on hpool or plots are steadily increasing, whilst others are increasing at a very frightening speed...

**Reply by dorkmo** [reply]
good points. It seems to me that the spikes down would have to be the result of people having issues keeping their farmers on, or smaller farmers just shutting down their computer for the night. There has to exist at least as many plots as the previous max. give or take depending on how the number is calculated. if you drew a line from the previous max to the latest max, the growth doesnt seem as alarming.

**Reply by dorkmo** [reply]
![image](https://user-images.githubusercontent.com/1923070/113922499-f5dcf600-97ac-11eb-9a45-5db12fe14a89.png)

for a visual. i drew a purple line for assumed network growth, and filled in pink where i assume existing plots are off network.

it looked like a lot of Mac users were having issues this week so maybe they're coming back.

also the chart is a bit deceptive itself. the total network scale on the left is by 10 units, while the right side is 2 units.

**Comment by GeorgePapageorgakis** [comment]
This is very wonderful experience farming XCH. almost 2 weeks with 2-3 days "Expected time to win" and 0 XCH farmed so far. GREAT! FEELS GOOD!

Seems legitimate right? Winner takes it all and the rest are rekt...
![εικόνα](https://user-images.githubusercontent.com/9392917/113162011-911f1a00-9247-11eb-8339-92695fd58744.png)

**Reply by PanzerKunst** [reply]
It's not winner takes all. As of today, 2657 addresses have won a block, and that number continues to increase. Maybe you are unlucky, or maybe your setup has an issue (such as an unsynced node or wallet).

If Chia frustrates you, it's probably better for you to look at alternatives. But the coin allocation works as intended, and it's fair: statistically equal to each farmer's percentage of the network.

**Comment by xLaBJup** [comment]
Mmmmh ....
[https://cointelegraph.com/news/chia-network-not-so-green-cpu-mining-reportedly-destroys-ssd-in-weeks](url)

Anyone any experience about that?

**Reply by hoffmang9** [reply]
Cheap SSDs are cheap... The sky is blue - https://github.com/Chia-Network/chia-blockchain/wiki/SSD-Endurance

**Reply by xLaBJup** [reply]
Yeah - let's just throw 'em to the trash and buy new ones .... 
But Chia is "A new green cryptocurrency" ....

**Reply by scotchegg** [reply]
Well you dont have to run them into the ground? Use then for 50 -70% of their life  only for plotting and you're likely still good to use them for regular use for many years.

**Comment by lixbai** [comment]
Hello, 
i am newbie from website

From my opinion, ONLY three types could earn money from chia.
One is chia team, two is the early farmer or called miner, three is the big one that have so much disk miner.

Today(2021-5-18 12:01:40) this is my chia software.
![today](https://user-images.githubusercontent.com/40021840/118596328-046ff180-b7de-11eb-9865-19e2bcafee64.PNG)

I dig it two days,  craete 18 plots, and get none reward. It shown that i need 2 years to estimated time to win a block. What the f...
I know i am late for the chia and i am not the big one....So ....dig chia block is no meanings for me.

BUT i has read the chia's doc and wiki, I found something confusion thing for me. 
The Chia said  "In Chia you can take some of your empty space, plot it, and win - probably the value of your computer - during that computer's life."
I think it is said for most personal computer. 
They said the Bitcoin is wasting much energy, chia will avoid much waste energy and good for environment than Bitcoin.  BUT we can find the wiki, they need State-of-the-art computer, CPU, Larger RAM, Larger SSD(NVME is best)....

AND the important is when we create the 101GB plot file and storege in Large disk, it's still wait the sync to mainnet and don't know it's wether will get reward or not. During in that time we couldn't delete the 101GB plot file. 
Because if we delete, we will be excluding get reward.
That's ridiculous!!!!
Some miner will buy more disk to storege the 101GB plot file, because they could't delete it! 
As the chia and time goes by, it need more and more disk, maybe it will use up all over the world disk oneday. I think it's not good for environment.

I think when Plot finished, it could keep an little file that include the result or they need something data, DO NOT keep a big file. In addition we could't delete it.

I think when the XCH price goes down or some miner couldn't afford the disk price, this mine type will be abandoned. Because fom an environmental and economic point of view, it is not sustainable!!!

Wish some miner get the reward convert to Bitcoin or some other coin early, DO not keep it.
Because the chia team still remain 21,000,000 XCH.......if they sell it on market, the XCH price will goes down fastly....
![today1](https://user-images.githubusercontent.com/40021840/118604233-7f89d580-b7e7-11eb-8668-721e7204ed88.PNG)


Conclusion:
From My opinion, If they don't solve need more disk to storeage more larger 101GB file, and need keep it and couldn't delete it. 
As the time goes by, the little miner and personal miner and single will stop the racing incease buy disk(becuase they will die before get enough reward to pay the disk fee.) and only remained the big miner that have much disk.......THIS IS Centralization!

This model is not sustainable!

Wish Chia team could think about this. if not, i think they are just play trick, no environment protection, no decentralization, no blockchain...
Just your money!

**Comment by OversizedWizard** [comment]
I have 178 plots showing time remaining to win 3 months 
I was 1 week when I started plotting. 
I've never won a reward!!!!

I have another 4rb to plot but I'm not keeping my hopes up for a win.

This seems to be worst than bitcoin for environmental impact.
After you plot, it uses small CPU power 

But, plotting takes  massive resources of hard drives and electric to plot the files only to have the whole network increase and time remaining to win go up!!!!!

Plots are passing filter but find nothing at all.

I've never won the lottery, so should we look at this the same???? Since it's like or describes as a lottery.

If that's the case, you could farm for a lifetime and never win.

This is not usefull for anyone new.

I started when network was I let 200Pb. Not it's 6.36 EiB.

Chia is dead before it begins properly for the normal average pc user.
It's even dead for me and I have 24tb storage available

**Reply by lixbai** [reply]
actually, i think it's a trick...
 chia team trick us.

**Reply by OversizedWizard** [reply]
Starting to think your right. Lol 
I'm gonna fill the remains drive I have and leave it as it is and see what happens. 
Like I said, I'm not keeping my hopes up.

I'll stick with my GPU mining.  At least that earns something daily lol

**Reply by lixbai** [reply]
i also think it worst than bitcoin for environmental impact.
because you couldn't delete the plot file, the big 101GB file...
if really that, it will be use up all over the world's disk may be oneday....
ahhh,,,, what ridiculous....

**Reply by MaximusChia** [reply]
it is nothing new.. Big whales will take more piece of the cake, consequently yo would need to join to a pool to balance the things.
Similar to bitcoin on its beggining. Its a lottery.
What makes the difference? you have to "mine" only "once"Plotting takes some time until is ready but its kept for indefinied time.. Instead of GPUs they have to keep hashing all the time, consuming a lot of energy and producing a lot of heat, heat is the enemy of electronics, this may wear out components very fast, PSU, motherboards included. ASICs also suffer, how much warranty Bitmain provides? isnt was only 90 Days? You may need change some components by the time, in this case the SSD. 
Then yo will need to invest on facilities, rooms with good cooling systems or ventilation. maybe soundproof envitroment too...
The energy used to hashing 6 GPU you can easily be farming a Houndred of HDD.
A drawback could be that, if your system dies while plotting, all that hours of working are lost!! This is sad, really.
Pools are a thing, sadly. Otherwise is a World lottery.
I have recently a week on this, testing my luck with 8TB...in a week will be full of plots. it does look very promising.

**Reply by OversizedWizard** [reply]
Good luck dude., I have 17.6 Tb at the moment and my rewards estimate is 3 months. Was 2 months 3 days ago and I'm still plotting everyday to fill my spare hard drives lol

**Comment by katmai** [comment]
you farm because of you see chia as one of the following options:
 
1 - you're a small guy and you don't have many resources, so that's a lottery. you may end up getting a winning ticket, that ends up paying for the cost of your hardware, which otherwise you wouldn't.

2 - hobby - you know that Bram Cohen built bittorrent, a technology that dramatically changed how people share data. you want to support the network because you want to see what this new technology will accomplish. You do not expect profits, but you'll be happy if they come.

3 - business - you see Chia as an investment, with huge potential in the future. You are willing to take a big risk, spend money on hard drives and hardware, building mining farms, spending your time improving efficiency, working with people and creating pools, building communities, pools, trying to contribute to the success of the project. High chances that you are going to make money, because you're willing to put in the effort, the time and the investment.

One can also spend their time arguing about decentralization vs rewards proportion, however it seems awkward that someone that is unwilling to contribute more, is debating and expecting to receive rewards proportional to people that do decide to put in the effort, and take the risks on an unproven technology. 

i understand that not everyone is unwilling, maybe a lot of people just don't have the resources, but you can't build a house with good intentions. Someone has to put in the time, the effort and the resources.

**Comment by DIlbert25** [comment]
Pure lucky:

With 8 plots I won 2 XCH.

With 20 plots I won 2 more XCH.

I already have 200 plots, and I didn't win any more ...

I will continue to plot, and participate in the game, until the launch of the pools ....

Not to mention that I use video card mining rigs to mine XCH.

**Comment by OversizedWizard** [comment]
You must have started early.  Mate has 18 coins. He had 144 plots when the network was only 207PiB in size .
He was getting a reward 2-3 days.
Now his estimate is 3 months just like mine at 178 plots.

I keep plotting more, but can't keep up to lower the estimated time.

Glad you managed to get something for yourself though👍
